<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>609</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа 3</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="label_0">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Выбор цвета</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" rowspan="19">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>731</width>
        <height>444</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QLabel" name="label_1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Цвет отрезков:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="lineColorButton">
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Цвет фона:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="fontColorButton">
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QLabel" name="label_8">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>16</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Выбор алгоритма</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="4">
     <widget class="QComboBox" name="comboBox">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>ЦДА</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Брезенхем (целые)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Брезенхем (действ.)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Брезенхем (с устр. ст.)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ву</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Библиотечный</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0" colspan="4">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>16</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Рисование отрезка</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>31</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>31</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>x0:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="3">
     <widget class="QLineEdit" name="lineEditX0">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_5">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>31</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>31</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>y0:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="3">
     <widget class="QLineEdit" name="lineEditY0">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_6">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>31</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>31</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>x1:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="3">
     <widget class="QLineEdit" name="lineEditX1">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_7">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>31</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>31</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>y1:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1" colspan="3">
     <widget class="QLineEdit" name="lineEditY1">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="4">
     <widget class="QPushButton" name="drawLineButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Нарисовать</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="4">
     <widget class="QLabel" name="label_9">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>16</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Рисование спектра</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="2">
     <widget class="QLabel" name="label_13">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>x:</string>
      </property>
     </widget>
    </item>
    <item row="12" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEditX">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="2">
     <widget class="QLabel" name="label_12">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>y:</string>
      </property>
     </widget>
    </item>
    <item row="13" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEditY">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="14" column="0" colspan="2">
     <widget class="QLabel" name="label_10">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Угол:</string>
      </property>
     </widget>
    </item>
    <item row="14" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEditStep">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="15" column="0" colspan="2">
     <widget class="QLabel" name="label_11">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Длина:</string>
      </property>
     </widget>
    </item>
    <item row="15" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEditLen">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="16" column="0" colspan="4">
     <widget class="QPushButton" name="drawLinesButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Нарисовать</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0" colspan="4">
     <widget class="QPushButton" name="analyseButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Анализ ступенчатости</string>
      </property>
     </widget>
    </item>
    <item row="18" column="0" colspan="4">
     <widget class="QPushButton" name="analyseButton_2">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Анализ времени</string>
      </property>
     </widget>
    </item>
    <item row="19" column="0" colspan="4">
     <widget class="QPushButton" name="discardButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Отменить</string>
      </property>
     </widget>
    </item>
    <item row="20" column="0" colspan="4">
     <widget class="QPushButton" name="discardAllButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Сброс</string>
      </property>
     </widget>
    </item>
    <item row="21" column="0" colspan="4">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>145</width>
        <height>149</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="22" column="0" colspan="4">
     <widget class="QPushButton" name="aboutButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>О программе</string>
      </property>
     </widget>
    </item>
    <item row="23" column="0" colspan="4">
     <widget class="QPushButton" name="authorButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Об авторе</string>
      </property>
     </widget>
    </item>
    <item row="24" column="0" colspan="4">
     <widget class="QPushButton" name="outButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>21</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
